src/Compiler/Checking/Expressions/CheckExpressions.fs

```diff
@@ -6003,10 +6003,10 @@ and TcExprUndelayed (cenv: cenv) (overallTy: OverallTy) env tpenv (synExpr: SynE
     | SynExpr.ImplicitZero m ->
         error(Error(FSComp.SR.tcConstructRequiresSequenceOrComputations(), m))
 
-    | SynExpr.DoBang (_, m)
-    | SynExpr.MatchBang (range = m)
```

Comment:
@psfinaki `{ MatchBangKeyword = m }` is what changes the range ?. I did nor meant to this as part of this PR.

Reply:
Yeah I guess it's here. 
I mean, maybe you can remove this particular change here then, if you're preparing a separate PR.

Reply:
Removed :)